### ISSUE TITLE
using pretrained=False has better performance

### ISSUE THREAD

**Issue by minooei** [insightfactory-app/ULMFIT-Persian#1]
Hi, thank you for sharing this code. 
here are some of my thoughts about it:
as suggested in this [link](https://medium.com/ai-quest/pre-trained-language-model-in-any-language-7531ea7217d4), for training for a language other than english we are better off not using the pre-trained model.
I tested with  pretrained=False option and in one epoch get better results.
epoch	train_loss	valid_loss	accuracy	time
0	3.935745	     3.842244	0.346726	1:04:21

and I think there is a typo in your code:
slice(51e-3,1e-2)
shouldn't it be slice(1e-3,1e-2) instead?

**Comment by puraminy** [comment]
yes `pretrained` must be set to False, I also noticed that problem in the code
